chapters/executors.rst

```diff
@@ -0,0 +1,113 @@
+==========================================
+A Guide to Jina Executors
+==========================================
+
+.. meta::
+   :description: A guide on Jina Executors
+   :keywords: Jina, Executors, Drivers
+
+.. note:: This guide assumes you have a basic understanding of Jina, if you haven't, please check out `Jina 101 <https://101.jina.ai>`_ first.
+
+.. contents:: Table of Contents
+    :depth: 2
+
+Motivation
+--------------------
+
+In Jina the Executors represent an algorithmic class and they are a key part to take the full potential that you can do with Jina. We have `several of them <https://docs.jina.ai/chapters/all_exec.html`_ and they all live in the `Hub <https://hub.jina.ai/#/home>`_, facilitating the work-flow while creating your app. Here we will see what exactly are they and how you can use them.
```

Comment:
In Jina the Executors represent an algorithmic class and they are a key part to take the full potential **of Jina**